examples/data-channels-create/main.go

```diff
@@ -16,7 +16,7 @@ import (
 func main() {
 	// Prepare the configuration
 	config := webrtc.RTCConfiguration{
-		ICEServers: []webrtc.RTCICEServer{
```

Comment:
Can we keep `ICE` instead of `Ice` 

Since it is an acronym I think it makes sense (and we won't break existing users)

Reply:
+1